### DISCUSSION TITLE
[question] How to display time with the logo summary option

### DISCUSSION THREAD

**Discussion by Katsec** [Yamato-Security/hayabusa#1311] [Q&A] [answered]
How to display time with the logo summary option?
<img width="1154" alt="image" src="https://github.com/Yamato-Security/hayabusa/assets/47964489/153bd371-816b-43a9-9158-7a1ea21b485e">

**Comment by YamatoSecurity** [comment, marked as answer]
Hi @Katsec 
Thank you for the question. Logon summary is a summary of logons so they do not include timestamps. If you want to see when a logon occurred, you can create a timeline with either `csv-timeline` or `json-timeline`.


If you have further questions, please post them to [Discussions](https://github.com/Yamato-Security/hayabusa/discussions) and not issues, as issues are for things like bugs, enhancements, etc...
